### ISSUE TITLE
Document how to load code in the editor

### ISSUE THREAD

**Issue by ISSOtm** [daid/rgbds-live#3]
Apparently you can pass URLs to text files, and I think base64 is an option too? Can this be documented in the README, so that it's easier to use rgbds-live externally & programmatically?

**Comment by avivace** [comment]
I agree.  RGBDS live already supports loading fileds from URLs (github gists)

Relevant part of the code: https://github.com/gbdev/rgbds-live/blob/master/www/js/storage.js#L32

**Comment by SelvinPL** [comment]
there are 3 options 
- `#https://gist.github.com/....` load all files from gist [example](https://gbdev.io/rgbds-live/?asm=-p+0xFF&link=-dt+-p+0xFF&fix=-jvc#https://gist.github.com/SelvinPL/0c418956154f06573a00fc23e499ba72)
- `#https://link/to/file/...` loads single file as `main.asm` [example](https://gbdev.io/rgbds-live/#https://gist.githubusercontent.com/SelvinPL/acd390e5c5d5ead9e37de86f86a3f104/raw/87d31e1bf119ec6c5ff37246f9fbe2b442712dcf/gb.asm)
- `#LZString.compressToEncodedURIComponent` - loads hash as compressed LZString and decompress usingLZString,decompressFromEncodedURIComponent [example](https://gbdev.io/rgbds-live/?fix=-C#LYQwlgdgdCDOwAACSA5AwgGQKoBECiABAEQAWIATgCYDuFAplJAMZEBQBrAynmgCpIB5FMQBqAIQA2ICAGsCSAEoBFIgBoCCgQFkADAG1UvAPoAJAIIocGPAqPiMFgNIBdVgEhydAC5h3XHvxCxCZ0IJR05Goa2voAJACMOjquBAQAVgAOBHgQXuQAngAKAPaQXqyslLAECQCsOgQAtAQAAuo6FTl5RaW5AFysANycvGZoju4SlAQkEuqxdEkdblMEsBnqs0OwXsWeBBhoOGteIF4ArrDsBKsg6nrkhzhoKQQZlyQEIABmQ3QQIAARhI6AQAG7A6RyRRKSbTO7yPAAMTsYgcKAmK0onweSDwznUIHcAA89l84TjyEgkQTyW46L43BBihkhrQwF4CN8yfYnNcyBJyqlmazBpQwLAgSD5MpUtcppS8bSiWKJVLQeL1lJ8vLpg8ni9CdtvARASAmDIAObkYrnCDTNAKMxaL6USieWDVXYEBrSab-dVfc67ZieYD-IU3eHqACkiSSRiWHVSqweYjQhU4yt1X3UAGZ4kM3h8CExihI9l7igRHc6OIMTntQWW7V4Iqw+sXYJ8fkMyxXyDdisUZNUJGAZKDyNPp0ZLfP50ZAcvl0ZiUNAcGvtUAEym-Jt6rEpcAFiXeaXO6X8SXOjnZ8tea+9oIlqvlpvlrv5DP5Av5Cvcgb3IZMCFJQciRTPVyHTQocGzVIMhZL5fkQkte1SWApHJTDsMgqMCDTDN4KNRDkIwghwiYHD0kHCAAC91D6Ro+zICBLVBEQnRdM1ZAIb1Cyg3MCEEgiHnEFxSII2Z5gATgADiSHNASYeY8xPDoG1gE0MhAEEvDbL4DPIMBN0PfjqyfCtqAiU0OWqYpviMvJTODUEwndOhPQ7AiEQka4-QION4lE1NZjABDKLoaimGuNI6MYghmOuKjTTihKmJYsisl7QYPRtfYngIAAKQKLnICACGKSrNQybUAEo5SE-UjkNGiBXKBAgA)

**Comment by avivace** [comment]
Should we have a mini rgbds-live website? or put this in some kind of help dialog?

**Comment by SelvinPL** [comment]
maybe redesign about menu to show a dialog with tabs like info uses help
